mws/mws.py

```diff
@@ -144,15 +144,21 @@ class MWS(object):
     # like "Merchant" in which case you define it like so.
     # ACCOUNT_TYPE = "Merchant"
     # Which is the name of the parameter for that specific account type.
+
+    # For using proxy you need to init this class with one more parameter proxies. It must look like 'ip_address:port'
+    # if proxy without auth and 'login:password@ip_address:port' if proxy with auth
+
     ACCOUNT_TYPE = "SellerId"
 
-    def __init__(self, access_key, secret_key, account_id, region='US', domain='', uri="", version="", auth_token=""):
+    def __init__(self, access_key, secret_key, account_id, region='US', domain='', uri="", version="", auth_token="",
+                 proxies=None):
```

Comment:
No, protocol is required, and I made http & https to avoid troubles with type of proxy, now this proxy settings works on my project, so this feature is tested for work